package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc -p tsconfig.json && npm run copy:files",
     "build:watch": "tsc -p tsconfig.json --watch && npm run copy:files",
-    "copy:files": "cp ./builders.json ./package.json dist/ && cp ./src/*.json dist/ && cp ./src/svg-icons-builder/*.json ./dist/svg-icons-builder/",
+    "copy:files": "cp ./src/builders.json ./package.json dist/ && cp ./src/*.json dist/ && cp ./src/svg-icons-builder/*.json ./dist/svg-icons-builder/",
```

I was looking at this too, because I had trouble with testability and working code. Along with this I have a PR that restores the basic test.

```suggestion
    "copy:files": "cp ./package.json dist/ && cp ./src/*.json dist/ && cp ./src/svg-icons-builder/*.json ./dist/svg-icons-builder/",
```

I believe ` cp ./src/*.json dist/` also covers the new location of builders.json.